test/release/examples/Makefile

```diff
@@ -26,13 +26,16 @@ clean: FORCE
 benchmarks: FORCE
 	cd benchmarks && $(MAKE)
 
+patterns: FORCE
```

Comment:
Could you please add the "patterns" directory to the "clean:" target in this Makefile?